examples/camel-example-debezium/src/main/java/org/apache/camel/example/debezium/DebeziumMySqlConsumerToKinesis.java

```diff
@@ -37,17 +36,18 @@
 
     private static final Logger LOG = LoggerFactory.getLogger(DebeziumMySqlConsumerToKinesis.class);
 
+    // use Camel Main to setup and run Camel
+    private static Main main = new Main();
+
     private DebeziumMySqlConsumerToKinesis() {
     }
 
     public static void main(String[] args) throws Exception {
 
         LOG.debug("About to run Debezium integration...");
 
-        final CamelContext camelContext = new DefaultCamelContext();
-
-
-        camelContext.addRoutes(new RouteBuilder() {
+        // add route
+        main.addRouteBuilder(new RouteBuilder() {
             public void configure() {
                 final PropertiesComponent pc = getContext().getComponent("properties", PropertiesComponent.class);
```

Comment:
camel-main has this out of the box, you dont need to setup the properties component for this